bazel-bin

```diff
@@ -0,0 +1 @@
+/home/orhanuysal/.cache/bazel/_bazel_orhanuysal/b8b367661f77e27a673f41c1d2c04ab5/execroot/__main__/bazel-out/k8-fastbuild/bin
```

Comment:
This file is bazel output and it doesn't need to be committed in repo. Please remove it